test/test_proxy.py

```diff
@@ -76,3 +76,21 @@ def test_ssl_context_with_proxy_is_respected(self):
             geocoder_dummy.geocode(self.remote_website_https)
         self.assertIn('SSL', str(cm.exception))
         self.assertEqual(1, len(self.proxy_server.requests))
+
+    def test_geocoder_constructor_uses_str_proxy(self):
+        base_http = urlopen(self.remote_website_http, timeout=self.timeout)
+        base_html = base_http.read()
+        geocoder_dummy = DummyGeocoder(proxies=self.proxy_url, scheme='http',
```

Comment:
Is there any reason to set the `scheme` kwarg explicitly? If not, could you please remove that?